.changeset/rich-pandas-collect.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/text': patch
+'@commercetools-uikit/utils': patch
```

Comment:
Shouldn't we migrate all packages to use this? 🤔

Reply:
Can also be a separate step and PR once this is in.

Reply:
Why migrate only the `Text` component then?

Reply:
yes.
Let's just focus on `Text` test and `invariant`, then we have address the basics before migrating everything :)